lib/repl.js

```diff
@@ -421,7 +421,10 @@ function REPLServer(prompt,
       }
     }
 
-    if (cmd || self.bufferedCommand) {
+    // self.context.setBreakpoint exists we are in the debugger.
```

Comment:
@bnoordhuis You're right, of course. I think have a better solution that I've just pushed. PTAL.